packages/webapp-libs/webapp-contentful/scripts/download-graphql-schema.js

```diff
@@ -0,0 +1,56 @@
+const { spawn } = require('child_process');
+const fs = require('fs-extra');
+const dotenv = require('dotenv');
+function runCommand(command, args) {
```

Comment:
We have this code in couple of places. WDYT about extract it to a single place?

Reply:
Yes and no. Those are very simple nodejs packages that do not require dependance on the whole typescript configuration (paths, pnpm workspace, etc) so it's not easy to create a reusable package. I decided it's not worth it.